Library/Homebrew/cmd/audit.rb

```diff
@@ -348,6 +348,7 @@ def audit_deps
         end
 
         dep.options.reject do |opt|
+          next true if opt.name == :devel
```

Comment:
Probably want to extend this to if it's `:devel` and the relevant dependency also defines a `devel` spec.